src/frontend/data/rootReducer.ts

```diff
@@ -0,0 +1,28 @@
+import { Reducer } from 'redux';
+
+import { appState } from '../types/AppData';
+import { modelName } from '../types/models';
+import { Video } from '../types/Video';
+import { Nullable } from '../utils/types';
+import { videos, VideosState } from './videos/reducer';
+
+export interface RootState {
+  context: {
+    jwt: Nullable<string>;
+    ltiResourceVideo: Nullable<Video>;
+    ltiState: appState;
+  };
+  resources: {
+    [modelName.VIDEOS]?: VideosState;
```

Comment:
This does not really reflect what is in the store. `resources.videos` always exists. Not typing it as optional will also save some check in `videoFormConnected` and `VideoPlayerConnected`